cli.js

```diff
@@ -21,6 +21,8 @@ async function main(argv) {
       Options
         --auth        [authentication]  (download) Authentication to access private repo
         --concurrency [number]          (download) How many parallel downloads should be used when downloading data (Default: 10)
+        --since       [buildNumber]     (download) Overrides the normal logic of which builds to download data for.
```

Comment:
Should we change this to something like: `Overrides the logic to get the build number to download the data from`

Reply:
Sure, if you suggest it as an edit, I can do it from my phone if you like. Otherwise will do when I get home

Reply:
I am merging the PR 🙂 I will include that with commit to publish new version